.gitignore

```diff
@@ -4,3 +4,5 @@
 /build/
 /dist/
 /.coverage
+
+tags
```

Comment:
?

Please leave your own tools out of my gitignore :)

Reply:
np, dropped the commit